client/src/component/index.js

```diff
@@ -1,4 +1,5 @@
 import Footer from './Footer/index';
 import InputField from './InputField/index';
+import NotificationCard from './NotificationCard';
 
-export default { InputField, Footer };
+export default { InputField, Footer, NotificationCard };
```

Comment:
because we export more than one value I can't use export.
```suggestion
export  { InputField, Footer, NotificationCard };
```
or you can use export default for one value 
```suggestion
export  { InputField as default, Footer, NotificationCard };
```

Reply:
done